### DISCUSSION TITLE
Display version number under Help About or Settings

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#82] [Feature requests] [answered]
I think there would be value in a page to display version number, compare to and display current version on web, show logo to alert that update available, and link to the main site page to download.  This is not the same as auto-update.  Just provides user ability to see their current version, as well as intuitive alert of new version available.  Same login as used to launch external browser, e.g. Chrome, Edge, Firefox, etc would then feed the user over to that page, etc...

**Comment by f1multiviewer** [comment, marked as answer]
<img width="301" alt="image" src="https://user-images.githubusercontent.com/106821994/176963721-75d5cfcd-89ed-4946-b0e5-bbd97f4b45dc.png">
Done! Ships next release :)
